resources/content/schema/admin/admin-auth.json

```diff
@@ -206,16 +293,31 @@
         "vnet.uri" : "cr",
 
         "volume" : "rd",
+        "volume.accountId" : "r",
+        "volume.allocationState" : "r",
+        "volume.attachedState" : "r",
         "volume.deviceNumber" : "r",
         "volume.format" : "r",
-        "volume.physicalSizeMb" : "",
         "volume.virtualSizeMb" : "r",
         "volume.uri" : "r",
         "volume.isHostPath" : "r",
+        "volume.instanceId" : "r",
+        
+      
+        "imageStoragePoolMap" : "r",
```

Comment:
I guess, it's when we were doing some hit and trials to get some tests working. Alphabetical will definitely be more neat. i'll shuffle them.